worlds/alttp/docs/multiworld_es.md

```diff
@@ -1,224 +1,120 @@
-   lanzar ficheros de ROM de SNES.
-    1. Extrae tu emulador al escritorio, o cualquier sitio que después recuerdes.
-    2. Haz click derecho en un fichero de ROM (ha de tener la extensión sfc) y selecciona **Abrir con...**
-    3. Marca la opción **Usar siempre esta aplicación para abrir los archivos .sfc**
-    4. Baja hasta el final de la lista y haz click en la opción **Buscar otra aplicación en el equipo** (Si usas Windows
-       10 es posible que debas hacer click en **Más aplicaciones**)
-    5. Busca el archivo .exe de tu emulador y haz click en **Abrir**. Este archivo debe estar en el directorio donde
-       extrajiste en el paso 1.
-
-### Instalación en Macintosh
-
-- ¡Necesitamos voluntarios para rellenar esta seccion! Contactad con **Farrak Kilhn** (en inglés) en Discord si queréis
-  ayudar.
-
-## Configurar tu archivo YAML
-
-### Que es un archivo YAML y por qué necesito uno?
-
-Tu archivo YAML contiene un conjunto de opciones de configuración que proveen al generador con información sobre como
-debe generar tu juego. Cada jugador en una partida de multiworld proveerá su propio fichero YAML. Esta configuración
-permite que cada jugador disfrute de una experiencia personalizada a su gusto, y cada jugador dentro de la misma partida
-de multiworld puede tener diferentes opciones.
-
-### Donde puedo obtener un fichero YAML?
-
-La página "[Generate Game](/games/A%20Link%20to%20the%20Past/player-options)" en el sitio web te permite configurar tu
-configuración personal y descargar un fichero "YAML".
-
-### Configuración YAML avanzada
-
-Una version mas avanzada del fichero Yaml puede ser creada usando la pagina
-["Weighted settings"](/games/A Link to the Past/weighted-options),
-la cual te permite tener almacenadas hasta 3 preajustes. La pagina "Weighted Settings" tiene muchas opciones
-representadas con controles deslizantes. Esto permite elegir cuan probable los valores de una categoría pueden ser
-elegidos sobre otros de la misma.
-
-Por ejemplo, imagina que el generador crea un cubo llamado "map_shuffle", y pone trozos de papel doblado en él por cada
-sub-opción. Ademas imaginemos que tu valor elegido para "on" es 20 y el elegido para "off" es 40.
-
-Por tanto, en este ejemplo, habrán 60 trozos de papel. 20 para "on" y 40 para "off". Cuando el generador esta decidiendo
-si activar o no "map shuffle" para tu partida, meterá la mano en el cubo y sacara un trozo de papel al azar. En este
-ejemplo, es mucho mas probable (2 de cada 3 veces (40/60)) que "map shuffle" esté desactivado.
+1. Descarga e instala (Archipelago)[https://github.com/ArchipelagoMW/Archipelago/releases/latest].
+   **El archivo del instalador se encuentra en la sección de assets al final de la información de version**.
+2. La primera vez que realices una generación local o parchees tu juego, se te pedirá que ubiques tu archivo ROM base. Este es tu archivo ROM de Link To THe Past japonés. Esto sólo debe hacerse una vez.
```

Comment:
```suggestion
2. La primera vez que realices una generación local o parchees tu juego, se te pedirá que ubiques tu archivo ROM base. Este es tu archivo ROM de Link to the Past japonés. Esto sólo debe hacerse una vez.
```

Reply:
Yeah thanks for this one, I was careless.